components/engine_rocks/src/table_properties.rs

```diff
@@ -123,3 +124,13 @@ impl DecodeProperties for RocksUserCollectedProperties {
             .ok_or(tikv_util::codec::Error::KeyNotFound)
     }
 }
+
+#[repr(transparent)]
+pub struct RocksUserCollectedPropertiesNoRc(rocksdb::UserCollectedProperties);
```

Comment:
```suggestion
pub struct RocksUserCollectedPropertiesNoRc(raw::UserCollectedProperties);
```
to be consistent.

but I don't see difference between `RocksUserCollectedProperties` and `RocksUserCollectedPropertiesNoRc`?

Reply:
`RocksUserCollectedProperties` is a ref-count struct to simplify lifetime code. However here rocksdb returns a reference, so we can only implement the trait for a new defined struct.

Reply:
`rocksdb::UserCollectedProperties` is different from `raw::UserCollectedProperties`. The latter is not returned by rocksdb `GetTableProperties` interface, so that we can't use it directly.

Reply:
how about change `raw::` to `raw_properties::` in this file for better readability?